logging/deployment/src/main/java/io/quarkiverse/googlecloudservices/logging/deployment/LoggingBuildSteps.java

```diff
@@ -0,0 +1,59 @@
+package io.quarkiverse.googlecloudservices.logging.deployment;
+
+import com.google.auth.oauth2.GoogleCredentials;
+
+import io.quarkiverse.googlecloudservices.logging.runtime.JsonFormatter;
+import io.quarkiverse.googlecloudservices.logging.runtime.LabelExtractor;
+import io.quarkiverse.googlecloudservices.logging.runtime.LoggingConfiguration;
+import io.quarkiverse.googlecloudservices.logging.runtime.TraceInfo;
+import io.quarkiverse.googlecloudservices.logging.runtime.TraceInfoExtractor;
+import io.quarkiverse.googlecloudservices.logging.runtime.ecs.EscJsonFormat;
+import io.quarkiverse.googlecloudservices.logging.runtime.format.JsonHandler;
+import io.quarkiverse.googlecloudservices.logging.runtime.format.TextHandler;
+import io.quarkiverse.googlecloudservices.logging.runtime.recorder.LoggingHandlerFactory;
+import io.quarkiverse.googlecloudservices.logging.runtime.util.LevelTransformer;
+import io.quarkiverse.googlecloudservices.logging.runtime.util.SimpleFormatter;
+import io.quarkiverse.googlecloudservices.logging.runtime.util.StackTraceArrayRenderer;
+import io.quarkus.arc.deployment.UnremovableBeanBuildItem;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.ExecutionTime;
+import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.deployment.builditem.LogHandlerBuildItem;
+
+public class LoggingBuildSteps {
+
+    private static final String FEATURE = "google-cloud-logging";
+
+    @BuildStep
+    public FeatureBuildItem feature() {
+        return new FeatureBuildItem(FEATURE);
+    }
+
+    @BuildStep
+    public UnremovableBeanBuildItem credentials() {
+        return UnremovableBeanBuildItem.beanTypes(GoogleCredentials.class);
+    }
+
+    @BuildStep
+    public UnremovableBeanBuildItem helperClasses() {
+        return UnremovableBeanBuildItem.beanTypes(
+                EscJsonFormat.class,
+                LevelTransformer.class,
+                JsonFormatter.class,
+                LabelExtractor.class,
+                LoggingConfiguration.class,
+                TraceInfoExtractor.class,
+                TraceInfo.class,
+                JsonHandler.class,
+                TextHandler.class,
+                SimpleFormatter.class,
+                StackTraceArrayRenderer.class);
+    }
```

Comment:
I don't understand why you make them unremovable as they are not CDI beans.

Reply:
I might have had problems with my local setup, because I did run into missing classes during testing. Will test again without them and see what happens.

Reply:
I've minimized it, but the interfaces the can be bound to the CDI context to customize the logging needed to stay (as did the configuration for some reason).